### PR TITLE
update - edit may slides

### PR DESCRIPTION
**Pull request by cgl**

### CODE REVIEW

File: organizers/2018-05-08.html [hunk 1]
```diff
@@ -84,15 +85,35 @@ <h2>Eating and Drinking</h2>
...
+
         <li><a href="https://www.djangoproject.com/weblog/">https://www.djangoproject.com/weblog/</a></li>
     </ul>
 </section>
 
+<section>
+    <h1>🎺 Python News</h1>
+    <h2>Release</h2>
+    <ul>
+        <li>Python 2.7.15 released last Tuesday</li>
```
Comment (adamchainz) [inline organizers/2018-05-08.html]: Not worth mentioning? No one uses 2.7 any more! It's 2018!

File: organizers/2018-05-08.html [hunk 2]
```diff
@@ -84,15 +85,35 @@ <h2>Eating and Drinking</h2>
...
+        <li>New PyPI launched</li>
+	<li>Legacy PyPI shutting down April 30!</li>
```
Comment (adamchainz) [inline organizers/2018-05-08.html]: indentation :)

File: organizers/2018-05-08.html (hunk 1 above)
Comment (cgl) [reply organizers/2018-05-08.html]: :) ok

Comment (adamchainz) [reply organizers/2018-05-08.html]: That was a bit of a troll, I bet you 50% of the meetup still use it, but perhaps we shouldn't encourage :P

Comment (cgl) [reply organizers/2018-05-08.html]: I heard that they drop 2.6 support. That was a big news for me but it was not on the news, I heard it in the weekly podcast I guess.

File: organizers/2018-05-08.html [hunk 3]
```diff
@@ -84,18 +85,36 @@ <h2>Eating and Drinking</h2>
...
+        <li><a href="https://ep2018.europython.eu/en/">EuroPython 2018</a>: <a href="https://blog.europython.eu/post/173666124852/europython-2018-call-for-proposals-cfp-is-open">Call for Proposals (CFP)</a> is open</li>
+        <li>The Python Software Foundation is fundraising! Help them reach their goal of $20K by May 12th!
+    </ul>
+</section>
+
+
 <section>
     <h1>Want to give a talk? 🤔</h1>
-    <p>Please come and chat with us!</p>
+    <p>Please come and chat with us! 👩🏻‍💻👨🏻‍💻</p>
```
Comment (adamchainz) [inline organizers/2018-05-08.html]: MOAR EMOJOYS

**Pull request merged**